### DISCUSSION TITLE
Issue with manual date entry in pnp search date range filter webpart

### DISCUSSION THREAD

**Discussion by tushar1989** [microsoft-search/pnp-modern-search#2872] [Q&A]
I am using PNP search filter webpart with date range filter with From and To date field , Is there a way if we can manually enter date in those fields, 
![image](https://user-images.githubusercontent.com/23469983/227490386-7d9df8f6-cdcd-4d9e-be24-9198123a1f69.png)

Thanks!

**Comment by wobba** [comment]
You can put the cursor in the box and start typing.
![image](https://user-images.githubusercontent.com/51104/229142422-a4590649-27dc-4864-951a-eef84095dbed.png)

**Reply by tushar1989** [reply]
@wobba thanks for update, we can update from date easily, we are not able to set cursor in  In second date picker 'To'  and enter date easily. Thanks!

**Reply by wobba** [reply]
Click twice.. works fine when I test.
